src/internal_modules/roc_fec/writer.cpp

```diff
@@ -92,12 +93,13 @@ bool Writer::resize(size_t sblen, size_t rblen) {
     return true;
 }
 
-void Writer::write(const packet::PacketPtr& pp) {
+status::StatusCode Writer::write(const packet::PacketPtr& pp) {
     roc_panic_if_not(is_valid());
     roc_panic_if_not(pp);
 
     if (!alive_) {
-        return;
+        // TODO: return StatusDead (gh-183)
+        return status::StatusNoData;
```

Comment:
Since StatusNoData is not appropriate for writers, and upper code is not ready yet for errors for writers, before 183 I think we should return StatusOK in cases where we don't fail currently.